### DISCUSSION TITLE
Arrival/departure boards at public transport platforms/stations

### DISCUSSION THREAD

**Discussion by NathanARF** [streetcomplete/StreetComplete#4819] [Ideas]
Hi. Sorry if this is in the wrong place, but I'd like to make a suggestion for a new quest, specifically relating to public transport platforms/stations that do not have the tag `departures_board=*` for arrivals and departures (https://wiki.openstreetmap.org/wiki/Key:departures_board). This could be for bus stops, railway stations, tram stops etc. This is an established tag and there are three main options that could be chosen by the user as the value: `timetable` (printed timetable at the stop), `realtime` (departures screen with live service information), and `no` for neither of the former two options. Is this something that could be implemented as a quest along the lines of "Does this stop have service or timetable information?"

**Comment by matkoniecz** [comment]
See https://github.com/streetcomplete/StreetComplete/issues/2263 https://github.com/streetcomplete/StreetComplete/issues/3553

not sure has situation changed since then
